cql/src/main/java/org/apache/cassandra/stargate/transport/internal/messages/AuthResponse.java

```diff
@@ -72,7 +72,11 @@ public AuthResponse(byte[] token) {
       byte[] challenge = negotiator.evaluateResponse(token);
       if (negotiator.isComplete()) {
         AuthenticatedUser authenticatedUser = negotiator.getAuthenticatedUser();
-        persistenceConnection().login(authenticatedUser);
+        if (!(authenticatedUser.isFromExternalAuth()
+            && Boolean.parseBoolean(
+                System.getProperty("stargate.cql_use_transitional_auth", "false")))) {
```

Comment:
Do we still need to check `stargate.cql_use_transitional_auth` now that we have it in `PlainTextTokenSaslNegotiator`